client/src/amplify-ui/block-switcher/block-switcher.style.ts

```diff
@@ -48,4 +48,8 @@ export const contentStyle = css`
       margin-bottom: 0;
     }
   }
+
+  .activeCodeBlock {
+    display: initial;
+  }
```

Comment:
Should this be in `block` CSS instead?

Reply:
Not sure if there's a file for that yet, but maybe there should be?

Reply:
🤷

I was in and out, down and dirty style. I leave it to you. To the extent that we don't plan to maintain this code much longer, query whether a 'slightly-hacky' approach might provide best ROI?